ironfish/src/rpc/routes/chain/followChainStream.ts

```diff
@@ -58,21 +51,15 @@ export const FollowChainStreamResponseSchema: yup.ObjectSchema<FollowChainStream
         sequence: yup.number().defined(),
       })
       .defined(),
-    block: yup
-      .object({
-        hash: yup.string().defined(),
-        sequence: yup.number().defined(),
-        previous: yup.string().defined(),
-        timestamp: yup.number().defined(),
-        graffiti: yup.string().defined(),
-        size: yup.number().defined(),
-        work: yup.string().defined(),
-        main: yup.boolean().defined(),
-        difficulty: yup.string().defined(),
-        noteSize: yup.number().nullable().defined(),
-        transactions: yup.array(RpcTransactionSchema).defined(),
-      })
-      .defined(),
+    block: RpcBlockHeaderSchema.concat(
+      yup
+        .object({
+          main: yup.boolean().defined(),
+          size: yup.number().defined(),
+          transactions: yup.array(RpcTransactionSchema).defined(),
+        })
```

Comment:
do we need to include `work` in the schema?

Reply:
i added work to the RpcBlockHeader object directly, we don't need to include it here